pages/deposits_and_withdrawals/cli_commands.md

```diff
@@ -0,0 +1,24 @@
+## CLI commands
+
+### Deposit to Subaccount
+
+```bash
+dydxprotocold tx sending deposit-to-subaccount <sender_key_or_address> <recipient_address> <recipient_subaccount_number> <quantums> [flags]
+```
+
+Example:
+```bash
+dydxprotocold tx sending deposit-to-subaccount dydx1g2ygh8ufgwwpg5clp2qh3tmcmlewuyt2z6px8k dydx1g2ygh8ufgwwpg5clp2qh3tmcmlewuyt2z6px8k 0 <usdc_quantum_uint64> --keyring-backend test --fees 5000adv4tnt
```

Comment:
nit: Since the fee denom (`adv4tnt` here) can change based on the network, might be better to preface this with
```
NATIVE_TOKEN_DENOM=adv4tnt
dydxprotocold tx sending deposit-to-subaccount dydx1g2ygh8ufgwwpg5clp2qh3tmcmlewuyt2z6px8k dydx1g2ygh8ufgwwpg5clp2qh3tmcmlewuyt2z6px8k 0 <usdc_quantum_uint64> --keyring-backend test --fees 5000$NATIVE_TOKEN_DENOM
```
or you can even reference the network constants doc [here](https://docs.dydx.exchange/networks/network1/network_constants#native-token-denom) in your doc.

This would make the instructions more chain-agnostic.

Reply:
Good call